projects/cashmere/src/lib/button/split-button/split-button.component.ts

```diff
@@ -34,6 +45,7 @@ export class SplitButtonComponent {
         this._tabIndex = value == null ? 0 : value;
     }
 
+    /* Theme color */
```

Comment:
Can we add the available options here? If possible, it might be preferable to use a unioned string literal of available options as the type so documents itself.